BAZProjectC1/Components/BasicInfoMovie/BasicInfoMovie.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  BasicInfoMovie.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 23/09/22.
+//
+
+import UIKit
+
+class BasicInfoMovie: UIView {
+    var viewBackgroundColor: UIColor? = .clear
```

Comment:
Esta propiedad puede ser privada
```suggestion
    private var viewBackgroundColor: UIColor? = .clear
```